cmd/avrogen/main_test.go

```diff
@@ -16,39 +16,39 @@ var update = flag.Bool("update", false, "Update golden files")
 
 func TestAvroGen_RequiredFlags(t *testing.T) {
 	tests := []struct {
-		name    string
-		args    []string
-		wantErr bool
+		name     string
+		args     []string
+		exitCode int
```

Comment:
Nit: it is typical to start test assertion properties with `want` to differentiate them from test input properties. I would suggest:
```suggestion
		wantExitCode int
```

Reply:
Doh. Apologies, had a brain glitch with that one. 🤦 
Fixed now!